### ISSUE TITLE
Calling meson on Windows

### ISSUE THREAD

**Issue by davidbrochart** [conda-forge/meson-feedstock#40]
Calling `meson` on Windows results in the following error:
```
'meson' is not recognized as an internal or external command,
operable program or batch file.
```
I tried different ways of calling `meson` in https://github.com/conda-forge/zchunk-feedstock/pull/3, but none of them worked.

**Comment by wolfv** [comment]
yes, I locally also had this problem :/

**Comment by pkgw** [comment]
I just merged #41. Let's see if the new build fixes this?

**Comment by wolfv** [comment]
I tested locally, and it should indeed fix it.
